benchmarks/README.md

```diff
@@ -5,6 +5,28 @@ The two main benchmarking scripts are
   - `result_analyzer.py` to aggregate the benchmark result in CSV form.
 
 
+## Reducing benchmark noise 
+
+It is important to keep the benchmark runs safe from external effects 
+to reduce noise. Run:
+
+```
+# Sets the CPU statically to the highest tuneable frequency. 
```

Comment:
nit: Maybe move the comments out of the code sections.

Reply:
Done.